#goal         
int64 vagon_type
string seat_sequence_type
int64 index
---
#result
string status
---
#feedback
float64 joint_info
